CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/ViewModels/ChatViewModel.cs

```diff
@@ -0,0 +1,43 @@
+﻿using CheckDrive.Mobile.Models;
+using System;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace CheckDrive.Mobile.ViewModels
+{
+    public class ChatViewModel : BaseViewModel
```

Comment:
maybe **NotificationViewModel** would be a better name?

Reply:
corrected